docs/docs/faq.mdx

```diff
@@ -138,6 +138,15 @@ To solely rely on Oh My Posh to set the prompt, you can configure the follwing s
 conda config --set changeps1 False
 ```
 
+### Python venv: Prompt is changed to display the environment name
+Upon activation, virtual environments created with `venv` change the prompt to add the activated environment's name.
+To disable this behaviour, set `VIRTUAL_ENV_DISABLE_PROMPT` environment variable to `1` on your system.
+You can add the following line to your Powershell profile:
+
+```powershell
```

Comment:
Do we have guides for other shells too? I know it's only changing the syntax, but could be handy?

Reply:
just updated with a new commit @JanDeDobbeleer